Lib/test/test_pathlib.py

```diff
@@ -1823,6 +1824,24 @@ def test_link_to(self):
         self.assertEqual(os.stat(r).st_size, size)
         self.assertTrue(q.stat)
 
+    @unittest.skipUnless(hasattr(os, "link"), "os.link() is not present")
+    def test_hardlink_to(self):
+        P = self.cls(BASE)
+        target = P / 'fileA'
+        size = target.stat().st_size
+        # linking to another path.
+        link = P / 'dirA' / 'fileAA'
+        link.hardlink_to(target)
+        self.assertEqual(link.stat().st_size, size)
+        self.assertTrue(os.path.samefile(target, link))
+        self.assertTrue(target.stat)
```

Comment:
I realize you mostly copied this test from above, but since `target.stat` is always a method, it will always be true. This seems to be a mistake in the tests.
(FTR, in the first iterations of the PEP design, `Path.stat` was a cached property, I think these checks are a remnant of this previous design)